cc-mt-service/openapi.yaml

```diff
@@ -0,0 +1,152 @@
+openapi: '3.0.2'
```

Comment:
backend and pk service design documents are using openapi 3.0.3, should all use the same version?